acra/configuring-maintaining/general-configuration/acra_server.md

```diff
@@ -0,0 +1,308 @@
+
+* `-ds`
+
+  Turn on HTTP debug server.
+  The server will be listening at `127.0.0.1:6060`.
+
+* `--dump_config`
+
+  Dump configuration to `configs/acra-server.yaml`.
+
+* `--encryptor_config_file=<filename>`
+
+  Path to Encryptor configuration file.
+  Default is empty.
+
+* `--generate_markdown_args_table`
+
+  Generate markdown file with text description of all flags.
+  Output file is `configs/markdown_acra-server.md`.
+  Works in pair with `--dump_config`.
+
+* `--http_api_enable={true|false}`
+
+  Enable HTTP API.
+
+* `--incoming_connection_api_port=<port>`
+
+  Port for AcraServer for HTTP API.
+  Default is `9090`.
+
+* `--incoming_connection_api_string=<url>`
+
+  Connection string for API like `tcp://x.x.x.x:yyyy` or `unix:///path/to/socket`.
+  Default is `tcp://0.0.0.0:9090/`.
+
+* `--incoming_connection_close_timeout=<seconds>`
+
+  Time that AcraServer will wait (in seconds) on restart before closing all connections.
+  Default is `10`.
+
+* `--incoming_connection_host=<host>`
+
+  Host for AcraServer to listen on.
+  Default is `0.0.0.0`.
+
+* `--incoming_connection_port=<port>`
+
+  Port for AcraServer to listen on.
+  Default is `9393`.
+
+* `--incoming_connection_prometheus_metrics_string=<url>`
+
+  URL which will be used to expose Prometheus metrics (use `<url>/metrics` address to pull metrics).
+  Default is empty.
+
+* `--incoming_connection_string=<url>`
+
+  Connection string like `tcp://x.x.x.x:yyyy` or `unix:///path/to/socket`.
+  Default is `tcp://0.0.0.0:9393/` (built from default host and port).
+
+* `--keys_dir=<path>`
+
+  Folder from which keys will be loaded.
+  Default is `.acrakeys`.
+
+* `--keystore_cache_size=<count>`
+
+  Count of keys that will be stored in in-memory LRU cache in encrypted form.
+  Use `0` to set unlimited size, `-1` to disable caching.
+  Default is `0`.
+
+* `--mysql_enable={true|false}`
+
+  Handle MySQL connections.
+  Default is `false`.
+
+* `--pgsql_escape_bytea={true|false}`
+
+  Escape format for Postgresql bytea data (**deprecated**, ignored).
+
+* `--pgsql_hex_bytea={true|false}`
+
+  Hex format for Postgresql bytea data (**deprecated**, ignored).
+
+* `--poison_detect_enable={true|false}`
+
+  Turn on poison record detection, if server shutdown is disabled, AcraServer logs the poison record detection and returns error.
+  Default is `true`.
+
+* `--poison_run_script_file=<filename>`
+
+  On detecting poison record: log about poison record detection, execute script, return decrypted data.
+  Default is empty (don't execute any script).
+
+* `--poison_shutdown_enable={true|false}`
+
+  On detecting poison record: log about poison record detection, stop and shutdown.
+  Default is `false`.
+
+* `--postgresql_enable={true|false}`
+
+  Handle Postgresql connections.
+  Default is `false`.
+
+* `--redis_db_keys=`
+
+  Number of Redis database for keys.
+  Default is `0`.
+  <!-- `acra-translator -help` says default is `-1` but in `cmd/redis.go` I see `redisDefaultDB = 0` -->
```

Comment:
hmm... it is interesting, why --help shows `-1` as default... I looked how these flags were declared and yes, used constant with value `0`...